nemo_text_processing/inverse_text_normalization/es_en/data/es_whitelist.tsv

```diff
@@ -1,16 +1,16 @@
-ud.	usted
```

Comment:
Same as previous comment for casing

Reply:
preferred output would be capitalized always